scprep/run/dyngen.py

```diff
@@ -0,0 +1,311 @@
+from . import r_function
+
+import pandas as pd
+
+_get_backbones = r_function.RFunction(
+    setup="""
+        library(dyngen)
+    """,
+    body="""
+        names(list_backbones())
+    """,
+)
+
+_DyngenSimulate = r_function.RFunction(
+    args="""
+        backbone_name=character(), num_cells=500, num_tfs=100, num_targets=50,
+        num_hks=25,simulation_census_interval=10, compute_cellwise_grn=FALSE,
+        compute_rna_velocity=FALSE, n_jobs=7, random_state=NA, verbose=TRUE
+    """,
+    setup="""
+        library(dyngen)
+    """,
+    body="""
+        if (!(backbone_name %in% names(list_backbones()))) {
+            stop("Input not in list of dyngen backbones.
```

Comment:
Can we throw this error in Python by using your `get_backbones` function so we get a cleaner error message?